docs/feature-comparison.md

```diff
@@ -0,0 +1,59 @@
+# Feature Comparison
+
+The following is an incomplete list of features differences between the legacy project system and the new project system. 
+
+**Feature**|**Legacy**|**New**|**Notes**
+---|---|---|---
+**Platforms**                                                               |
+.NET Standard                                                               | No       | Yes
+.NET Core                                                                   | No       | Yes
+.NET Framework                                                              | Yes      | Partial  | No designer/AppModel support for new project system
+**App Models**                                                              |
+ASP.NET Core (.NET Framework & .NET Core)                                   | No       | Yes
+ASP.NET                                                                     | Yes      | No
+Xamarin                                                                     | Yes      | No
+Universal Windows Platform (UWP)                                            | Yes      | No
+Windows Presentation Framework (WPF)                                        | Yes      | No
+Windows Forms                                                               | Yes      | No
+**Build**|
+Target multiple frameworks (multi-target) from single project               | No       | Yes
+Show build (design-time) errors & warnings in Error List as you make them   | No       | Yes
+**Debug**|
+Debug multiple frameworks from single project                               | No       | Yes
+Debug with multiple environments from single project ("launch profiles")    | No       | Yes
+Debug settings persistence                                                  |project.csproj.user (per user, per machine)|launchsettings.json (source control)
```

Comment:
Settings persistence in project file (source control) is supported by csproj ([example](https://github.com/tunnelvisionlabs/InheritanceMargin/blob/1e6a7ef714cf9295c77cb1ab76b962d98d777160/Tvl.VisualStudio.InheritanceMargin/Tvl.VisualStudio.InheritanceMargin.csproj#L26-L31))